tests/rptest/tests/topic_recovery_test.py

```diff
@@ -1481,29 +1481,29 @@ def all_replicas_in_sync(topic, *, partition, num_replicas):
 
         def verify():
             num_leaders = 0
-            try:
-                for topic in recovered_topics:
-                    topic_state = self.rpk.describe_topic(topic.name)
-                    # Describe topics only works after leader election succeded.
-                    # We can use it to wait until the recovery is completed.
-                    for partition in topic_state:
-                        self.logger.info(f"partition: {partition}")
-                        if partition.leader in partition.replicas:
-                            num_leaders += 1
-
-                            # If we have a leader, we can check if all replicas are in sync
-                            if not all_replicas_in_sync(
-                                    topic.name,
-                                    partition=partition,
-                                    num_replicas=len(partition.replicas)):
-                                self.logger.debug(
-                                    "partition replicas are not in sync yet")
-                                return False
-            except:
-                return False
+            for topic in recovered_topics:
+                topic_state = self.rpk.describe_topic(topic.name)
+                # Describe topics only works after leader election succeded.
+                # We can use it to wait until the recovery is completed.
+                for partition in topic_state:
+                    self.logger.info(f"partition: {partition}")
+                    if partition.leader in partition.replicas:
+                        num_leaders += 1
+
+                        # If we have a leader, we can check if all replicas are in sync
+                        if not all_replicas_in_sync(topic.name,
+                                                    partition=partition.id,
```

Comment:
the commit looks like it should just be a whitespace change to remove indentation, but this line was `partition=partition` and now it is `partition.id`.

Reply:
🦅 👁️

Reply:
Updated. TIL that even in this case github won't drop the reviews/approvals as the "cumulative" diff didn't change. Cool. https://github.com/redpanda-data/redpanda/compare/bca41536164b3c6d6e1f8fad17aee473b33e265a..f9477eb612e6a13edbcdb79a8b93c6b02f92eaf5